### DISCUSSION TITLE
Question with background UI and Dialogic

### DISCUSSION THREAD

**Discussion by maron5466** [dialogic-godot/dialogic#2324] [General]
Hi, I am using Godot 4.2 and Dialogic version alpha 14.
I am trying to have on my scene a clickable sprite2D (I was planning to use a 2d area for the mouse button input, showing a new panel over Dialogic timelines and everything else), while Godot timelines are running. But I do not understand Dialogic 2 enough to do it. Can anybody give me advice?

**Comment by Jowan-Spooner** [comment]
For anything that should be "above" dialogic, refer to Canvas Layers, see this: https://docs.dialogic.pro/faq.html#i-cannot-see-or-press-my-buttons

**Reply by maron5466** [reply]
Thank you. I am sorry If I ask again, but I have a problem with loading. In general, it saves and loads perfectly every moment of any timeline. But, if I load from the main menu, the game tries to overimpose the Dialogic timeline 
over the main menu, rather than sending me to the scene. I tried to use
if global.citta == "Delos":
		get_tree().change_scene_to_file("res://delo.tscn")   
but by doing this the game starts another series of Dialogic timelines, which are triggered by the func ready of the original scene after the games change scenes. Do you have any possible solution?
